Source/HelixToolkit.Wpf/Importers/StudioReader.cs

```diff
@@ -547,7 +547,7 @@ private void ReadMaterial(BinaryReader reader, int chunkSize)
                         }
 
                         mg.Children.Add(new SpecularMaterial(new SolidColorBrush(specular), specularPower));
-
+                        mg.Children.Add(new EmissiveMaterial((new SolidColorBrush( luminance))));

```

Comment:
Is this needed?
Is it a defect or enhancement?
Can we see this effect on any of the included models? (/Models/3ds)